DEVELOPMENT.md

```diff
@@ -70,72 +70,78 @@
         name = "google-cloud-showcase-v1beta1-java",
         deps = [
             # This is the new microgen target above.
-            ":showcase_java_gapic",
+            "//:showcase_java_gapic",
             # The following targets already exist.
-            ":showcase_java_grpc",
-            ":showcase_java_proto",
-            ":showcase_proto",
+            "//:showcase_java_grpc",
+            "//:showcase_java_proto",
+            "//:showcase_proto",
         ],
     )
     ```
 
 4.  Build the new target.
 
     ```sh
-    bazel build google/showcase/v1beta1:showcase_java_gapic
+    bazel build //google/showcase/v1beta1:showcase_java_gapic
     ```
 
 ## Code Formatting
 
 -   Run linter checks without actually doing the formatting.
 
     ```sh
-    bazel run :google_java_format_verification
+    bazel run //:google_java_format_verification
     ```
 
 -   Format files.
 
     ```sh
-    bazel run :google_java_format
+    bazel run //:google_java_format
     ```
 
 ## Test Running
 
--   Run all unit tests.
+-   Run all unit and integration tests.
 
     ```sh
     bazel test //...
     ```
 
--   Run a single unit test like `JavaCodeGeneratorTest.java`
+-   Run all unit tests.
 
     ```sh
-    bazel run //src/test/java/com/google/api/generator/engine:JavaCodeGeneratorTest
+    bazel test $(bazel query //... | grep ^//:unit_)
     ```
 
--   Run a single integration test for API like `Redis`, it generates Java source
-    code using the Java microgenerator and compares them with the goldens files
-    in `test/integration/goldens/redis`.
+-   Run a single unit test like `JavaCodeGeneratorTest.java`
 
     ```sh
-    bazel test //test/integration:redis
+    bazel test //:unit_com_google_api_generator_engine_JavaCodeGeneratorTest
```

Comment:
minor: can you please instead of using `unit_com_google_api_generator_engine_JavaCodeGeneratorTest` for samples here (and below) use `GrpcServiceStubClassComposerTest` for example, as this one is very likely to change (and need these commands to be executed) while JavaCodeGenerator is simply never touched as it is a very "fake" test (tests that geneator can generate generic java classes, outside of gapic context).

Reply:
> for samples here (and below) use `GrpcServiceStubClassComposerTest` for example,

The PR is doing that. Please take a look at `[GOLDEN_UPDATING_UNIT_TESTS in BUILD.bazel](https://github.com/googleapis/gapic-generator-java/pull/919/files#diff-7fc57714ef13c3325ce2a1130202edced92fcccc0c6db34a72f7b57f60d552a3R152-R171). There's no change regarding the test names. I think you were confused with this particular example `JavaCodeGeneratorTest`. The old setup was doing the same.